src/platform/export/fileConverter.ts

```diff
@@ -111,7 +111,7 @@ export class FileConverter implements IFileConverter {
             // As far as this method is concerned the export was successful, whether the user opens the file or not
             // should not have any bearing on the completion (resolving) this method.
             // Hence don't await.
-            this.openExportedFile(format, target).catch(noop);
+            await this.openExportedFile(format, target).catch(noop);
```

Comment:
Do we want to let this throw if it fails?

Reply:
> hus the tests might fail as they are checking activeTextEditor for validation.

Not true, we wait for actext text editor to change, instead of checking if its different, hence even if the file opens 2 seconds later we're still waiting for that (i.e. its an async wait, instead of a sync check for active editor)